sriovnet_test.go

```diff
@@ -0,0 +1,86 @@
+package sriovnet
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+
+	utilfs "github.com/Mellanox/sriovnet/pkg/utils/filesystem"
+)
+
+
+func setupGetNetDevicesFromPciEnv(t *testing.T, pciAddress string, deviceNames []string) func() {
+	var err error
+	utilfs.Fs = utilfs.NewFakeFs()
+	pciNetDir := filepath.Join(PciSysDir, pciAddress, "net")
+	err = utilfs.Fs.MkdirAll(pciNetDir, os.FileMode(0755))
```

Comment:
maybe create net dir only if deviceNames is not empty

that way you can support both PCI device with and without net devices

Reply:
could you elaborate please?